shared/src/errors/errors.ts

```diff
@@ -60,16 +68,91 @@ export const errors = {
       new BadRequestError(
         `Aucun des signataires n'a signé la convention avec l'identifiant ${conventionId}.`,
       ),
+    missingActor: ({
+      conventionId,
+      role,
+    }: { conventionId: ConventionId; role: Role }) =>
+      new BadRequestError(`There is no ${role} on convention ${conventionId}.`),
+    unsupportedRoleRenewMagicLink: ({ role }: { role: Role }) =>
+      new BadRequestError(
+        `Le rôle ${role} n'est pas supporté pour le renouvellement de lien magique.`,
+      ),
   },
   establishment: {
     notFound: ({ siret }: { siret: SiretDto }) =>
       new NotFoundError(
         `Aucune entreprise trouvée avec le siret : ${siret}. Êtes-vous sûr d'avoir bien tapé votre siret ?`,
       ),
+    siretMismatch: () =>
+      new ForbiddenError("Siret mismatch in JWT payload and form"),
     conflictError: ({ siret }: { siret: SiretDto }) =>
       new ConflictError(
         `Une entreprise avec le siret ${siret} existe déjà dans notre annuaire.`,
       ),
+    contactRequestConflict: ({
+      siret,
+      appellationCode,
+      minimumNumberOfDaysBetweenSimilarContactRequests,
+    }: {
+      siret: SiretDto;
+      appellationCode: AppellationCode;
+      minimumNumberOfDaysBetweenSimilarContactRequests: number;
+    }) =>
+      new ConflictError(
+        [
+          `A contact request already exists for siret ${siret} and appellation ${appellationCode}, and this potential beneficiary email.`,
+          `Minimum ${minimumNumberOfDaysBetweenSimilarContactRequests} days between two similar contact requests.`,
+        ].join("\n"),
+      ),
+    contactRequestContactModeMismatch: ({
+      siret,
+      contactMethods,
+    }: {
+      siret: SiretDto;
+      contactMethods: { inParams: ContactMethod; inRepo: ContactMethod };
+    }) =>
+      new BadRequestError(
+        `Contact mode mismatch: ${contactMethods.inParams} in params. In contact (fetched with siret ${siret}) : ${contactMethods.inRepo}`,
+      ),
+    immersionOfferNotFound: ({
+      siret,
+      appellationCode,
+    }: {
+      siret: SiretDto;
+      appellationCode: AppellationCode;
+    }) =>
+      new NotFoundError(
+        `Establishment with siret '${siret}' doesn't have an immersion offer with appellation code '${appellationCode}'.`,
+      ),
+    immersionOfferBadRequest: ({
```

Comment:
idem , chaud pour virer le `immersion` 
```suggestion
    offerBadRequest: ({
```